### PR TITLE
Homepage impact header

### PR DESCRIPTION
**Pull request by weerd**
### What's this PR do?

This pull request wraps up the last big item for the new Homepage! 🎉 It tackles finishing up fine-tuning the markup and styles for the impact header.

Responsive example:
Homepage Impact Header

### How should this be reviewed?

👀 

### Any background context you want to provide?

Twas a bit tricky but a fun challenge!

### Relevant tickets

References [Pivotal #171056723](https://www.pivotaltracker.com/story/show/171056723).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Added GIF of front-end changes on small, medium, and large screens.

### CODE REVIEW

File: resources/assets/components/pages/HomePage/NewHomePage.js [hunk 1]
```diff
@@ -129,40 +134,55 @@ const NewHomePageTemplate = ({ articles, campaigns, title }) => {
...
-              </div>
+                <div className="px-4 py-5 text-center">
+                  <p className="font-league-gothic leading-none mb-1 text-5xl text-teal-300 uppercase">
+                    3.7 million
+                  </p>
+                  <p className="font-bold m-0 mb-2 text-lg text-white uppercase">
+                    Cigarette Butts Collected
+                  </p>
+                  <p className="italic m-0 text-white">Get the Filter Out</p>
+                </div>
```
[the remark this answers is not in the document]
Comment (DFurnes) [reply resources/assets/components/pages/HomePage/NewHomePage.js]: (And if you did, totally fine for it to be inline in this file, imho!)

File: resources/assets/components/pages/HomePage/NewHomePage.js [hunk 2]
```diff
@@ -129,40 +134,55 @@ const NewHomePageTemplate = ({ articles, campaigns, title }) => {
...
+              className="base-12-grid"
               css={css`
+                margin-top: -250px;
+                padding-bottom: 0;
+                padding-top: 0;
+
                 @media (min-width: ${tailwindScreens.md}) {
-                  display: grid;
-                  grid-template-columns: 1fr 1fr 1fr;
+                  margin-top: -104px;
```
[the remark this answers is not in the document]
Comment (weerd) [reply resources/assets/components/pages/HomePage/NewHomePage.js]: Not sure... but it's the same number as the extra padding in the element with the background to allow for the overlap. Seemed magic number-y to me, but more-or-less how it was in the mocks. cc @lkpttn

File: resources/assets/components/pages/HomePage/NewHomePage.js [hunk 3]
```diff
@@ -129,40 +134,55 @@ const NewHomePageTemplate = ({ articles, campaigns, title }) => {
...
-                <p className="italic m-0 text-white">Teens for Jeans</p>
-              </div>
+              <div
+                className="bg-blurple-500 py-4 grid-wide"
+                css={css`
+                  @media (min-width: ${tailwindScreens.xl}) {
+                    display: grid;
+                    grid-template-columns: 1fr 1fr 1fr;
+                  }
+                `}
```
[the remark this answers is not in the document]
Comment (weerd) [reply resources/assets/components/pages/HomePage/NewHomePage.js]: Haha, I keep forgetting they're available now. I think this was from when I had done it a while back before your original suggestion to switch to the classes, so can defs update!

File: resources/assets/components/pages/HomePage/NewHomePage.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (weerd) [reply resources/assets/components/pages/HomePage/NewHomePage.js]: Can defs change to a nice round `100px` :)

**Pull request merged**